app/controllers/sessions_controller.rb

```diff
@@ -82,7 +82,7 @@ def do_login
       if status.success?
         StatsD.increment "login.success"
         set_login_flash
-        redirect_back_or(url_after_create)
```

Comment:
should this be `redirect_back_or_to` to avoid a change in behavior?

Reply:
Good question. I tried, but some tests failed. I wasn't sure if that is related to the way we setup test or not. It failed with 404 on GET "/session" if I remember well. It is probably related to different implementation. On Clearance side original URL is [stored only for GET requests](https://github.com/thoughtbot/clearance/blob/8d663ad4ecb26bba824c1b2b2cffa2bafc6ffa34/lib/clearance/authorization.rb#L74-L76).